### DISCUSSION TITLE
Whisper-Live

### DISCUSSION THREAD

**Discussion by Dimlight** [openai/whisper#306] [Show and tell]
Is it possible to write a code for live automatic transcription like the following link: https://github.com/oliverguhr/wav2vec2-live? 

It would be a great contribution.

**Comment by qunash** [comment]
I made a naive implementation which feeds audio from mic every 1 sec. It works okay.
https://huggingface.co/spaces/anzorq/openai_whisper_stt

**Reply by qunash** [reply]
I should add that this demo runs on a CPU, so it's super slow.
It's preferred to copy the code and run it on a GPU in a colab or on your local machine.

**Reply by olivMertens** [reply]
So next text incoming :) thanks

**Comment by olivMertens** [comment]
Try this morning,with english spoken and french, did'nt work...

**Comment by turnkit** [comment]
saw this today: https://github.com/shirayu/whispering

**Comment by zoq** [comment]
We worked on a live transcription, including a browser extension - https://github.com/collabora/WhisperLive

Also https://col.la/transcription has a demo for transcription, summary, and translation that is based on Whisper.
